cbw_api_toolbox/cbw_api.py

```diff
@@ -71,6 +89,7 @@ def _request(self, verb, payloads, body_params=None):
             self.logger.error("An error occurred, please check your API_URL.")
             sys.exit(-1)
 
+        return None
```

Comment:
I added a return statement here because pylint was complaining, but maybe it is resolved in an other PR ?